geometry_msgs/Pose pose
uint32 id